software/winrm/src/main/java/org/apache/brooklyn/util/core/internal/winrm/winrm4j/Winrm4jTool.java

```diff
@@ -184,7 +191,13 @@ public Winrm4jTool(ConfigBag config) {
     }
 
     private io.cloudsoft.winrm4j.winrm.WinRmTool connect() {
-        return io.cloudsoft.winrm4j.winrm.WinRmTool.connect(host+":"+port, user, password);
+        WinRmTool.Builder builder = WinRmTool.Builder.builder(host, user, password);
+        builder.setAuthenticationScheme(authenticationScheme);
+        builder.useHttps(useSecureWinrm);
+        if (useSecureWinrm) {
+            builder.disableCertificateChecks(true);
```

Comment:
This deserves a TODO in the code, I think. That we disable certificate checks when told to "use secure" is concerning, but I understand why you do it (see new FIXME in `WinRmMachineLocation.java`).